spec/presenters/summary/sections/employment_details_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe Summary::Sections::EmploymentDetails do
+  subject { described_class.new(crime_application) }
+
+  let(:crime_application) do
+    instance_double(
+      CrimeApplication,
+      to_param: '12345',
+      income: income,
+    )
+  end
+
+  let(:income) do
+    instance_double(
+      Income,
+      lost_job_in_custody: 'yes',
+      date_job_lost: Date.new(2023, 11, 20),
+    )
+  end
+
+  describe '#name' do
+    it { expect(subject.name).to eq(:employment_details) }
+  end
+
+  describe '#show?' do
+    context 'when there is no income_details' do
+      let(:income) { nil }
+
+      it 'shows this section' do
+        expect(subject.show?).to be(false)
+      end
```

Comment:
Is this test incorrectly named?

Reply:
I renamed AND reordered (just FYI in case it causes confusion)